index.test.js

```diff
@@ -1,21 +1,34 @@
 // Testing checkTaskOff()
 
-// comment out everything from checkTaskOff() on line 72 in app.js apart from [return e;]  
+// comment out everything from checkTaskOff() on line 76(before addition to code it was line 72) in app.js apart from [return e;]  
 // test("checkTaskOff() should receive an object as argument", () => {
 //   let result = checkTaskOff({});
 //   equal(typeof result, 'object', `checkTaskOff() should receive an object as argument, received: ${typeof result}`);
 // })
 
 
-test("checkTaskOff() should change target parent's class", () => {
-  const firstNote = document.querySelectorAll(".tasks__note")[0];
-  const noteCheckBtn = firstNote.firstElementChild;
+/*test("checkTaskOff() should change target parent's class", () => {
+    const firstNote = document.querySelectorAll(".tasks__note")[0];
+    const noteCheckBtn = firstNote.firstElementChild;
```

Comment:
firstNote.firsElementChild was selecting the check button, which was the first element child, but after adding the delete button in the HTML it became the 3rd node, I believe that's where the issue was 😉 I will change it to firstNote.childNodes[3] and it should work 👍🏻